src/psyclone/tests/psyGen_test.py

```diff
@@ -2693,6 +2757,21 @@ def test_binaryoperation_can_be_printed():
     assert "BinaryOperation[operator:'+']\n" in str(binaryOp)
 
 
+def test_binaryoperation_gen_c_code():
+    '''Test that a BinaryOperation node can generate its C representation'''
+
+    biOp = BinaryOperation("+")
```

Comment:
pylint doesn't like this variable name :-)

Reply:
s/biOp/binary_operation